### PR TITLE
feat: use nextjs.js standalone output for improvised docker image

### PR DESCRIPTION
**Pull request by G3root**

### CODE REVIEW

**Comment by Mythie** [comment]
Hey Nafees, I'd like to come back to this one. I've noticed a few changes to the container that I'm not sure I agree with, is there any reason we can't just use the older Dockerfile with the updated next config?

**Comment by G3root** [comment]
> Hey Nafees, I'd like to come back to this one. I've noticed a few changes to the container that I'm not sure I agree with, is there any reason we can't just use the older Dockerfile with the updated next config?

the older docker file is just fine, but it has a few issues, such as packages are installed twice and final size of the image is `2.08GB`.  whereas the standalone output results in an image size of only `295MB`. inorder to use the standalone build we have to copy some files from `.next/static` and `.next/standalone` folder manually, so older docker file cannot be used.

File: packages/prisma/package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "format": "prisma format",
     "prisma:generate": "prisma generate",
     "prisma:migrate-dev": "prisma migrate dev",
-    "prisma:migrate-deploy": "prisma migrate deploy"
+    "prisma:migrate-deploy": "prisma migrate deploy",
+    "post-install": "prisma generate"
```
[the remark this answers is not in the document]
Comment (G3root) [reply packages/prisma/package.json:13]: Thanks for pointing it out , this was committed by mistake

**Comment by Mythie** [comment]
> > Hey Nafees, I'd like to come back to this one. I've noticed a few changes to the container that I'm not sure I agree with, is there any reason we can't just use the older Dockerfile with the updated next config?
> 
> the older docker file is just fine, but it has a few issues, such as packages are installed twice and final size of the image is `2.08GB`. whereas the standalone output results in an image size of only `295MB`. inorder to use the standalone build we have to copy some files from `.next/static` and `.next/standalone` folder manually, so older docker file cannot be used.

Apologies for the delay once again, I corrected the parts I wasn't so agreeable on and we're in a good state now!

Awesome work, we appreciate it 🫶🏻

/tip 50

**Pull request merged**